tms/clickup/urls.py

```diff
@@ -3,5 +3,9 @@
 
 urlpatterns = [
     path("register/", views.SingUpView.as_view(), name="register"),
-    path("login/",views.LoginView.as_view(),name="login"),
+    path("login/", views.LoginView.as_view(), name="login"),
+    path("create_project/", views.ProjectCreateView.as_view(), name="Create_project"),
```

Comment:
for the create and list of project u can use one url and create and listing both are work. don't need to write create and list different urls